frictionless/metadata.py

```diff
@@ -174,6 +174,8 @@ def from_descriptor(
         # TODO: remove in next version
         # Transform with a base class in case the type is not available
         cls.metadata_transform(descriptor)
+        # Set type to string for missing field type
+        descriptor = cls.metadata_schema_default(descriptor)
```

Comment:
@shashigharti 
Have you tried just updating `Field.metadata_profile` and adding a default value for the `type` on the Field class? I'm not sure but it might work. Can you please check?

Reply:
I tried second one it did not work but haven't tried the first one. I will recheck both.